app/src/main/kotlin/com/intive/picover/profile/viewmodel/ProfileViewModel.kt

```diff
@@ -61,7 +59,7 @@ class ProfileViewModel @Inject constructor(
 		}
 	}
 
-	fun updateName(name: String) {
+	fun updateName(name: String = username.value) {
```

Comment:
is the default value needed?

Reply:
It's needed for tests to pass. I'm trying to remove it.

Reply:
I want to change
```
ManageProfileParam(
    { authRepository.updateUserName("Marian K") },
    { tested.updateName("Marian K") },
),
```
to
```
ManageProfileParam(
    { authRepository.updateUserName(any()) },
    { tested.updateName() },
),
```
but I have no idea how to change `profileMethod` in `ManageProfileParam` to be able to pass `any()` inside. I was trying to make it an extension on `MockKMatcherScope` but it doesn't work. Can anybody help me with this?

Reply:
@michal-kucznerowicz Notice that most of the tests from `ProfileViewModelTest` are throwing an exception under the hood, even if they are green. That could be a clue :)

Reply:
Done, default value removed